astropy/coordinates/tests/test_regression.py

```diff
@@ -16,6 +16,7 @@
 from ...time import Time
 
 from ...tests.helper import pytest, assert_quantity_allclose
+from .test_matching import HAS_SCIPY, OLDER_SCIPY
```

Comment:
Add ``# pylint: disable=W0611`` to make sure these don't get removed by accident again in future.